tests/TestCli.cpp

```diff
@@ -575,7 +579,7 @@ void TestCli::testCreate()
     QVERIFY(db2);
 
     // Testing with existing keyfile
-    QString databaseFilename3 = testDir->path() + "testCreate3.kdbx";
+    QString databaseFilename3 = testDir->path() + "/testCreate3.kdbx";

```

Comment:
@droidmonkey :interrobang: how was it working without the extra slash?

Reply:
It was creating a file in the temp directory, not in the temporary directory made from this class.